pkg/runner/cluster_k8s.go

```diff
@@ -692,6 +699,27 @@ func (c *ClusterK8sRunner) createTestplanPod(ctx context.Context, podName string
 			},
 			RestartPolicy: v1.RestartPolicyNever,
 			InitContainers: []v1.Container{
+				{
+					Name:            "wait-for-sidecar",
+					Image:           "busybox",
+					ImagePullPolicy: v1.PullIfNotPresent,
+					Args:            []string{"-c", "until nc -vz $HOST_IP 6060; do echo \"Waiting for local sidecar to listen to $HOST_IP:6060\"; sleep 2; done;"},
```

Comment:
There might be a state in which the TCP socket is open, but still unhealthy -- i.e. missing permissions -- maybe wait until healthy?

Reply:
Missing permissions? Then everything would be failing, and that would be a misconfiguration on the environment.

Ideally we want to have an endpoint that says `sidecar is healthy` - but this is rather difficult - we'd have to run healthchecks on the CNI plugins as well. I think for now making sure that the process is running is good enough.